lib/miq_pglogical/connection_handling.rb

```diff
@@ -2,10 +2,37 @@ class MiqPglogical
   module ConnectionHandling
     extend ActiveSupport::Concern
 
+    included do
+      delegate :logical_replication_supported?, :to => :class
+    end
+
     module ClassMethods
+      def logical_replication_supported?
+        return @logical_replication_supported if defined?(@logical_replication_supported)
+
+        is_superuser = ActiveRecord::Base.connection.select_value("SELECT usesuper FROM pg_user WHERE usename = CURRENT_USER;", "SQL")
```

Comment:
Minor, and I commented with this in the other comment, but it was easily missed in the diff.  You don't need the trailing `;` and I also don't think you need the second parameter with the `SQL`.